tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+	"include": [
+		"source"
+	],
+	"compilerOptions": {
+		"jsx": "react",
+		"jsxFactory": "h",
+		"sourceMap": true,
+		"target": "esnext",
+		"allowSyntheticDefaultImports": true, // Allow `import` of CJS modules
+		// Search under node_modules for non-relative imports.
+		// "moduleResolution": "node",
+		// Process & infer types from .js files.
+		"allowJs": true,
+		// // Don't emit; allow Babel to transform files.
+		// "noEmit": true,
+		// // Enable strictest settings like strictNullChecks & noImplicitAny.
+		// "strict": true,
+		// // Disallow features that require cross-file information for emit.
+		// "isolatedModules": true,
+		// // Import non-ES modules as default imports.
+		// "esModuleInterop": true
+	},
+}
```

Comment:
Eventually, it would be nice to use https://github.com/sindresorhus/tsconfig to get the best strict defaults. We would still have to override some things though.